### ISSUE TITLE
CS1591: triggers for code generated by svcutil.

### ISSUE THREAD

**Issue by jimmymain** [DotNetAnalyzers/StyleCopAnalyzers#1410]
Hi,

code generated by svcutil is being flagged as needing XML documentation on public members.
this occurs even through 'suppress results from managed code' is flagged in the project options.

`ArrayOfDateTime` is flagged as needing documentation.

```
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//     Runtime Version:4.0.30319.34209
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------

namespace X.Y.Z
{
    using System.Runtime.Serialization;
    
    
    [System.Diagnostics.DebuggerStepThroughAttribute()]
    [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Runtime.Serialization", "4.0.0.0")]
    [System.Runtime.Serialization.CollectionDataContractAttribute(Name="ArrayOfDateTime", Namespace="http://tempuri.org/", ItemName="dateTime")]
    public class ArrayOfDateTime : System.Collections.Generic.List<System.DateTime>
    {
```

**Comment by sharwell** [comment]
Hi @jimmymain,

CS1591 is a warning produced by the C# compiler (based on the `CS` prefix). You can avoid this warning by one of the following methods:

1. By generating your service interfaces/classes as `internal` instead of `public`.
1. By using a `[SuppressWarnings]` attribute (in a separate file so it's not overwritten when the code is regenerated).
1. By disabling or reducing the severity of the warning as described in my comment in #1371.

**Comment by jimmymain** [comment]
Hi Sam,

Thanks, I got confused because I turned on analysis, and stylecop at the
same time. Sorry for the trouble.
